src/networking.c

```diff
@@ -1690,10 +1690,15 @@ void acceptCommonHandler(connection *conn, struct ClientFlags flags, char *ip) {
     client *c;
     UNUSED(ip);
 
-    char addr[NET_ADDR_STR_LEN] = {0};
-    char laddr[NET_ADDR_STR_LEN] = {0};
-    connFormatAddr(conn, addr, sizeof(addr), 1);
-    connFormatAddr(conn, laddr, sizeof(addr), 0);
```

Comment:
this used to always return /unixsocket for unix socket client, we can keep it if it does not important.
```
    } else if (sa.ss_family == AF_UNIX) {
        if (ip) {
            int res = snprintf(ip, ip_len, "/unixsocket");
            if (res < 0 || (unsigned int)res >= ip_len) goto error;
        }
        if (port) *port = 0;
```